src/components/vm/consoles/vnc.jsx

```diff
@@ -48,6 +55,79 @@ const Enum = {
     KEY_DELETE: 111,
 };
 
+export const VncState = ({ vm, vnc }) => {
+    const Dialogs = useDialogs();
+
+    function add_vnc() {
+        Dialogs.show(<AddVNC idPrefix="add-vnc" vm={vm} />);
+    }
+
+    function edit_vnc() {
+        Dialogs.show(<EditVNCModal idPrefix="edit-vnc" vm={vm} consoleDetail={vnc} />);
+    }
+
+    if (vm.state == "running" && !vnc) {
+        return (
+            <EmptyState>
+                <EmptyStateBody>
+                    {_("Graphical support not enabled.")}
+                </EmptyStateBody>
+                <EmptyStateFooter>
+                    <Button variant="secondary" onClick={add_vnc}>
+                        {_("Add VNC")}
```

Comment:
The text here should be consistent. I.e. "VNC support not enabled." above and "Add VNC support" below.

Reply:
Yes, that's precisely my point -- we have spice as well, and strings should be consistent.